crates/pipeline_manager/src/config.rs

```diff
@@ -262,7 +290,10 @@ impl ManagerConfig {
         }
     }
 
-    /// Directory where the manager generates Rust crate for the project.
+    /// Directory where the manager generates the rust crate for the project.
+    ///
+    /// e.g., `$HOME/.dbsp/cargo_workspace/
```

Comment:
Maybe just say `<working-dir>/xyz` for these examples instead of $HOME/.dbsp?

Reply:
I was under the impression `$HOME/.dbsp` is the default so that's why I put that

Reply:
The working directory is a configuration parameter for the pipeline manager. In the container setup we use something like /working-dir.